File: TypeScript/codewars/Typescript/L1-Bartender-drinks.ts
export function getDrinkByProfession(profession: string): string {
	const drinks: Record<string, string> = {
		jabroni: "Patron Tequila",
		"school counselor": "Anything with Alcohol",
		programmer: "Hipster Craft Beer",
		"bike gang member": "Moonshine",
		politician: "Your tax dollars",
		rapper: "Cristal",
	};

	return drinks[profession.toLocaleLowerCase()] ?? "Beer";
}

File: Csharp/CodeWars/src/Completed/L1BartenderDrinks.cs
namespace CodeWars.Completed;

public class L1BartenderDrinks
{
    public static string GetDrinkByProfession(string p)
    {
        Dictionary<string, string> drinks = new()
        {
            { "jabroni", "Patron Tequila" },
            { "school counselor", "Anything with Alcohol" },
            { "programmer", "Hipster Craft Beer" },
            { "bike gang member", "Moonshine" },
            { "politician", "Your tax dollars" },
            { "rapper", "Cristal" }
        };

        return drinks.ContainsKey(p.ToLower()) ? drinks[p.ToLower()] : "Beer";
    }
}
